### DISCUSSION TITLE
Discord url expired

### DISCUSSION THREAD

**Discussion by castaway** [fleaflet/flutter_map#1935] [General]
Could someone update it please? (Assuming still in use)

**Comment by josxha** [comment]
@castaway what invite link are you using? the invite link on the readme and docs does not expire
https://github.com/fleaflet/flutter_map?tab=readme-ov-file#get-help

**Comment by JaffaKetchup** [comment]
Hi @castaway,
The link should be this one, which has not expired: https://discord.com/invite/BwpEsjqMAH

If you saw a different link, please let me know where!

If this link still doesn't work, please let me know your Discord username.

**Reply by castaway** [reply]
All the links I found on docs/pub.dev errored .. this one looks quite similar, and yet works! No idea whats going on, thanks!
